components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/util/APIClient.java

```diff
@@ -163,7 +204,8 @@ private void handleSuccess(ActionInvocationResponse.Builder builder, HttpEntity
         try {
             builder.response(handleSuccessResponse(entity));
         } catch (ActionInvocationException e) {
-            builder.errorLog("Unexpected response for status code: " + statusCode + ". " + e.getMessage());
+            builder.errorLog("Unexpected error occured on action execution response for status code " + statusCode
+                    + ". " + e.getMessage());
```

Comment:
This should say unexpected response for status code 200. Here the problem is the expected response is not received for the defined status code

Reply:
Updated in